website/docs/guides/provider.md

```diff
@@ -0,0 +1,38 @@
+---
+id: provider
+title: Manage providers
+sidebar_position: 1
+---
+### Manage Providers
+
+### Before you start 
+You can manage a provider assigned to a specific project.
+Firstly you need to be registered to the accounting-system and then you need to contact the administrator of the project , that this provider is associated, in order to assign you one or more roles, on the provider.
```

Comment:
You can manage a provider assigned to a specific project.
Firstly, you need to be registered to the accounting-system and then you need to contact the administrator of the project, that this provider is associated with, to assign you one or more roles on the provider.